.circleci/config.yml

```diff
@@ -38,4 +38,32 @@ jobs:
           destination: reports
       - store_artifacts:
           path: build/reports/jacoco/jacocoFullReport/
+  push_translations:
```

Comment:
Can you limit this job only to `master` and `develop` branches?

There is no point to do it for PR branches.

Reply:
Done.

But we have disabled the option _Build forked pull requests_, so we don't worry about third parts PRs.

Reply:
Still, for a example, I can add some translation in my PR and then remove it, so better to have it enabled just for `master` and `develop` branches.